src/components/Avatar.js

```diff
@@ -5,5 +5,9 @@
 export default function Avatar(props) {    
     const { person } = props;
 
-    // return ...
+    return (
+    <div>
+        <img class="Avatar" src={person.imageUrl}></img>
```

Comment:
- there is no class attribute in React, see requirements
- missing alt attribute: We use alt attribute in img element to make them accessible, they should have a meaningful value which represents the image
- no need to add divisions here - avoid creating unnecessary div's in your projects